.vscode/extension.json

```diff
@@ -0,0 +1,7 @@
+{
+    "recommendations": [
+      "dbaeumer.vscode-eslint",
```

Comment:
`formulahendry.auto-rename-tag` 
`xabikos.javascriptsnippets`
`streetsidesoftware.code-spell-checker`

mentioning some more extensions that I use generally

Reply:
okay cool 👍🏻